packages/claim-cli/package.json

```
@@ -0,0 +1,71 @@
+{
+	"name": "@liskhq/claim-cli",
+	"version": "0.1.0",
+	"private": true,
+	"description": "CLI to claim process",
+	"author": "Lisk Foundation <admin@lisk.com>, lightcurve GmbH <admin@lightcurve.io>",
+	"license": "Apache-2.0",
+	"keywords": [
+		"blockchain",
+		"lisk"
+	],
+	"main": "index.js",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/LiskHQ/lisk-token-claim.git"
+	},
+	"bugs": {
+		"url": "https://github.com/LiskHQ/lisk-token-claim/issues"
+	},
+	"homepage": "https://github.com/LiskHQ/lisk-token-claim#readme",
+	"bin": {
+		"claim-cli": "./bin/run"
+	},
+	"engines": {
+		"node": ">=18.0.0"
+	},
+	"scripts": {
+		"build": "shx rm -rf dist tsconfig.tsbuildinfo && tsc -b",
+		"postpack": "shx rm -f oclif.manifest.json",
+		"prepack": "yarn build && oclif manifest && oclif readme",
+		"prepare": "yarn build",
+		"test": "mocha --forbid-only \"test/**/*.test.ts\""
+	},
+	"oclif": {
+		"bin": "claim-cli",
+		"dirname": "claim-cli",
+		"commands": "./dist/commands",
+		"plugins": [
+			"@oclif/plugin-help"
+		],
+		"topicSeparator": " "
+	},
+	"dependencies": {
+		"@inquirer/prompts": "^5.0.4",
+		"@liskhq/lisk-codec": "^0.5.0",
+		"@liskhq/lisk-cryptography": "4.1.0",
+		"@liskhq/lisk-db": "^0.3.10",
```

Not important but may be better to add exact version
```suggestion
		"@liskhq/lisk-codec": "0.5.0",
		"@liskhq/lisk-cryptography": "4.1.0",
		"@liskhq/lisk-db": "0.3.10",
```